src/common-elements/fields.ts

```diff
@@ -97,9 +97,11 @@ export const RecursiveLabel = styled(FieldLabel)`
 
 export const PatternLabel = styled(FieldLabel)`
   color: #0e7c86;
+  font-family: ${props => props.theme.typography.code.fontFamily};
+  font-size: 12px;
   &::before,
   &::after {
-    font-weight: bold;
+    content: ' ';
```

Comment:
The intention here is to ensure the pattern string is adequately spaced from adjacent inline elements.

Without it, there can be cases (see the before screenshot) in which there's no spacing at all.